app/controllers/application_controller.rb

```diff
@@ -74,9 +75,10 @@ def metrics_logger
   end
 
   def ensure_visit_integrity
-    unless visit && visit.prisoner && visit.prisoner.prison_name.present?
+    unless visit && visit.prisoner && visit.prison_name.present?
       redirect_to edit_prisoner_details_path,
-        notice: 'You need to complete missing information to start or continue your visit request'
+        notice: 'You need to complete missing information to start' \
```

Comment:
I think this is a prime candidate for pulling out into the locale file, rather than just wrapping the line.

We automated localised notices in peoplefinder; have a look at [application controller](https://github.com/ministryofjustice/peoplefinder/blob/5c8aa8daf3e95bc198fd3f4e22bb3d6ad541f10e/app/controllers/application_controller.rb#L58-L71) and [people controller](https://github.com/ministryofjustice/peoplefinder/blob/5c8aa8daf3e95bc198fd3f4e22bb3d6ad541f10e/app/controllers/people_controller.rb#L62) if you're interested in doing something similar.